iceprod/server/data/www/rest.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Helper functions that use REST API
+ */
+
+async function fetch_json(url, passkey, json=null) {
+	try {
+		var payload = {headers: new Headers({'Authorization': passkey})};
+		if (json) {
+			payload['body'] = JSON.stringify(json);
+			payload['method'] = 'PUT';
```

Comment:
Also need to set the header: `'Content-Type': 'application/json'`

Reply:
https://github.com/WIPACrepo/iceprod/commit/c83ad9db666da980c107eb5aa457c2a19d4cf9c5

Reply:
Actually, sorry, should that be in the header?

Reply:
yes, it's in the headers dict

Reply:
5977009